### PR TITLE
Added Finnish translation.

### PR DESCRIPTION
**Pull request by lcrosenbu**

### CODE REVIEW

**Comment by rodrigo-brito** [comment]
Please, don't translate the aplication name.

File: app/src/main/res/values-fi/strings.xml [hunk 1]
```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <string name="app_name">Fire TODOList</string>
```
Comment (lcrosenbu) [inline app/src/main/res/values-fi/strings.xml]: The application name was left untranslated.

Comment (rodrigo-brito) [inline app/src/main/res/values-fi/strings.xml:3]: The field "description" has the aplication name. Please, don't translate the app name in this sentence.

**Comment by lcrosenbu** [comment]
Updated.

**Comment by rodrigo-brito** [comment]
Thank you so much!

**Pull request merged**